addons/mail/static/src/components/attachment_link_preview/attachment_link_preview.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.AttachmentLinkPreview" owl="1">
+        <div class="o_AttachmentLinkPreview" t-att-class="{'o-isCompact': props.isCompact}">
+            <t t-if="attachmentLinkPreview and attachmentLinkPreview.attachment">
```

Comment:
attachment should be required on link preview, no need for guard on it

Reply:
Without the guard, I get an error when I delete a link preview in the composer

Reply:
then you don't have the field required and/or the inverse is lacking isCausal